v2/proto/claim.proto

```diff
@@ -11,53 +11,734 @@ message Claim {
 
 message Channel {
     bytes public_key = 1;
+
     string title = 2;
     string description = 3;
-    string contact_email = 4;
-    string homepage_url = 5;
-    repeated string tags = 6;
+    string thumbnail_url = 4;
+    repeated string tags = 5;
+    repeated Language languages = 6;
```

Comment:
I imagine that the languages and locations lists are specific to the content in the particular stream claim.

Since channels also have `languages` you could release a movie in multiple languages by making a channel for it (listing all languages in the channel metadata) then publish a version of the movie in those languages in that channel (and list only the single language in the stream claim).

Perhaps channels within channels could also help here.